### ISSUE TITLE
Neural Networks with no input (dummy) to optimise transformed output?

### ISSUE THREAD

**Issue by jordan-bird** [keras-team/keras#12524]
Hello, 

I would like to create either a) a deep dense neural net that has n outputs with no activation function (for example, 3 numbers output) or b) an RNN that internally performs the same based on observed effects. So, I want to say, feed it stationary values of X=1 Y=1 but reconsider what the loss value is for the three inputs, in order to train to optimise my function

My issue is that I need to transform the output before the loss is given, eg:
`loss my_function=(a, b, c)`

Where: 

- Loss is the _true loss_ of the network
- my_function accepts the three outputs of the network at each step (a, b and c), rounded to integers
- my_function returns a float between 0.0 and 100.0 where 0.0 is the worst and 100.0 is the best possible value

Is it possible to get the network to learn from the above my_function as true loss? I've been trying for days to implement it. I can return the value of my_function() from my code, but when I input this code inside a custom loss function it just executes once at the start (TF Graphs).

Is there any way I can replace my loss with the above function using a Lambda layer, custom loss function or even a callback? I'm not sure how to do this, or if it's even possible (?)

This question seems to be aiming for a somewhat similar goal: https://github.com/keras-team/keras/issues/2691
but unfortunately I'm not quite good enough at Keras (yet!) to apply it to my problem

Help would be much appreciated,
Thanks

PS: I know a genetic or PSO would probably be better but my project is to show the effects of each method with results on multiple problems
